src/libs/ReportUtils.js

```diff
@@ -363,7 +363,7 @@ function isUserCreatedPolicyRoom(report) {
 /**
  * Whether the provided report is a Policy Expense chat.
  * @param {Object} report
- * @param {String} report.chatType
+ * @param {String} [report.chatType]
```

Comment:
Same as before

Reply:
@kubabutkiewicz is necessary to avoid errors in this JS file.